CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 0.6.3dev
 
 ## 0.6.2 (2023-03-05)
-
+* [Feature] Adds sql magic test to list of possible magics to test datasets
```

Comment:
Since the latest version is `## 0.6.5dev`, you should:
1. Merge the latest master code with ploomber/jupysql
2. Move this `* [Feature] Adds sql magic test to list of possible magics to test datasets` into `## 0.6.5dev`